model/lib/user/user_profile.dart

```diff
@@ -1,15 +0,0 @@
-import 'package:equatable/equatable.dart';
-
-class UserProfile with EquatableMixin {
```

Comment:
We should not delete it. Maybe we should add more information in user profile in the future (avatar, info, ...)

Reply:
IMO, It's just too redundant at this point. We always have to take care of it. We can create a separate object if we have to use it later. On the other hand we already have `PersonalAccount` which is an object that contains most of the current user's information.